lib/logidze/has_logidze.rb

```diff
@@ -13,6 +13,10 @@ module ClassMethods # :nodoc:
       def has_logidze
         include Logidze::Model
       end
+
+      def has_logidze?
+        included_modules.include? Logidze::Model
```

Comment:
Mmm, I thought about that, but classes without `has_logidze` wouldn't respond to this method. So we would have to check it beforehand, which not very convenient. Am I missing something?